x-pack/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/components/calendars/description_dst.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC, PropsWithChildren } from 'react';
+import React, { memo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiDescribedFormGroup, EuiFormRow, EuiLink } from '@elastic/eui';
+import { useMlKibana } from '../../../../../../../../../contexts/kibana';
+
+export const DescriptionDst: FC<PropsWithChildren<unknown>> = memo(({ children }) => {
+  const {
+    services: { docLinks },
+  } = useMlKibana();
+  const docsUrl = docLinks.links.ml.calendars;
+  const title = i18n.translate(
+    'xpack.ml.newJob.wizard.jobDetailsStep.additionalSection.calendarsDstSelection.title',
+    {
+      defaultMessage: 'DST Calendars',
+    }
+  );
+  return (
+    <EuiDescribedFormGroup
+      title={<h3>{title}</h3>}
+      description={
+        <FormattedMessage
+          id="xpack.ml.newJob.wizard.jobDetailsStep.additionalSection.calendarsDstSelection.description"
+          defaultMessage="Something about DST calendars. {learnMoreLink}"
```

Comment:
@szabosteve can you help me with the text for this section which appears in the AD job wizard?
![image](https://github.com/user-attachments/assets/767eec89-d7ef-4052-9faf-b4a593d80c8d)